### DISCUSSION TITLE
Experimental version updates

### DISCUSSION THREAD

**Discussion by dbryant69** [flybywiresim/aircraft#4711] [Help] [answered]
I haven't seen an update of the experimental version for a few days now. My version is c43d026. is this the latest or am I missing some. I used to get an update every day or so, but now the installer is just saying that the experimental version is installed.
Am I on the latest version please?

**Comment by MapleNAV-TCASFAULT** [comment, marked as answer]
If you are not on our discord server already, read the latest server announcement. It gives insight on why the experimental version has not updated recently. Long story short; there will be a new experimental version and they need to tune it until the branch can be updated.

**Comment by tracernz** [comment]
https://github.com/flybywiresim/a32nx/blob/master/README.md#experimental-branch-build
